posthog/api/insight.py

```diff
@@ -161,6 +167,8 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                     queryset = queryset.filter(Q(saved=False))
             elif key == "user":
                 queryset = queryset.filter(created_by=request.user)
+            elif key == "favorited":
+                queryset = queryset.filter(Q(favorited=True))
```

Comment:
Isn't that the same with fetching non-saved insights currently? We can always come back to fetching non-saved and non-favorited insights if we build a use case for it? 😄 Also, which `person_property.py` file are you referring to? I can't seem to find it

Reply:
Makes sense! Sorry I actually meant `person_definition.py` (don't know where the person_property came from haha)

Reply:
I don't see that file either :(

Reply:
https://github.com/PostHog/posthog/blob/e48bd8b08b67bcf7036bcbe6ad247b26180b139d/posthog/api/property_definition.py#L40

But not a big deal, it can be an improvement for later

Reply:
i will definitely add this as an improvement for next PR